src/Main.elm

```diff
@@ -376,6 +381,35 @@ view model =
         NotFoundRoute ->
             viewError "Not Found"
 
+---- SUBSCRIPTIONS ----
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    let
+        comicNumbers =
+            RemoteData.map2 (,)
+                (RemoteData.map .number model.latest)
+                (RemoteData.map .number model.requested)
+
+        handleKeyDown code =
+            case ( code, comicNumbers ) of
+                ( 37, RemoteData.Success ( _, requestedNumber ) ) ->
+                    if requestedNumber /= 1 then
+                        LoadRequestedComic <| requestedNumber - 1
+                    else
+                        Noop
+
+                ( 39, RemoteData.Success ( latestNumber, requestedNumber ) ) ->
+                    if requestedNumber /= latestNumber then
+                        LoadRequestedComic <| requestedNumber + 1
```

Comment:
Similar logic exists in `viewNavigation` function to get the previous and next comic links. It would be great if you can extract it into smaller functions and reuse at both the places.

Reply:
Can we also show navigation buttons as follows just to help the visitors that it exists?

![xkcd comic browser 2017-12-31 18-08-39](https://user-images.githubusercontent.com/876195/34461652-c0820666-ee55-11e7-88e3-f7db20fcb9ef.png)

Here's the modified markup it needs:

```diff
 <div class="navigation">
-  <a href="#/comics/1014">Previous</a>
-  <a href="#">Latest</a>
-  <a href="#/comics/random">Random</a>
-  <a href="#/comics/1016">Next</a>
+  <a href="#/comics/1014" style="text-decoration: none;text-align: center;display: inline-block;vertical-align: middle;">
+    <span style="text-decoration: underline;">Prev</span>
+    <br>
+    <code style="font-family: monospace;border: 1px solid #000;border-radius: 5px;padding: 0 5px 2px;color: #fff;background: #000;">←</code>
+  </a>
+  <a href="#" style="text-decoration: none;text-align: center;display: inline-block;vertical-align: middle;">
+    <span style="text-decoration: underline;">Latest</span>
+    <br>
+    <code style="font-family: monospace;border: 1px solid #000;border-radius: 5px;padding: 0 5px 2px;color: #fff;background: #000;">H</code>
+  </a>
+  <a href="#/comics/random" style="text-decoration: none;text-align: center;display: inline-block;vertical-align: middle;">
+    <span style="text-decoration: underline;">Random</span>
+    <br>
+    <code style="font-family: monospace;border: 1px solid #000;border-radius: 5px;padding: 0 5px 2px;color: #fff;background: #000;">R</code>
+  </a>
+  <a href="#/comics/1016" style="text-decoration: none;text-align: center;display: inline-block;vertical-align: middle;">
+    <span style="text-decoration: underline;">Next</span>
+    <br>
+    <code style="font-family: monospace;border: 1px solid #000;border-radius: 5px;padding: 0 5px 2px;color: #fff;background: #000;">→</code>
+  </a>
 </div>
```

We will need to map `h/H` keypress to `Latest` and `r/R` keypress to `Random` link.